zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -70,6 +71,24 @@
     private final PreparedStatement selectTraceDurationTail;
     private final Map<String,String> metadata;
 
+    private final ThreadLocal<Set<String>> writtenNames = new ThreadLocal<Set<String>>() {
+                private final AtomicLong hour
```

Comment:
fixed with https://github.com/thelastpickle/zipkin/commit/dd3ba49169298eabc9a90ed342de0080ba1de114
(i'll squash that commit into this PR's one commit)